recovery/elliptics_recovery/types/merge.py

```diff
@@ -686,6 +683,98 @@ def succeeded(self):
         return self.result
 
 
+class ServerSendRecovery(object):
```

Comment:
This class doesn't update any statistics, so it is impossible to track recovery progress.